pkg/extensions/sync/utils.go

```diff
@@ -165,3 +166,9 @@ func pushSyncedLocalImage(repo, tag, uuid string,
 
 	return nil
 }
+
+// sync needs transport to be stripped to not be wrongly interpreted as an image reference
+// at a non-fully qualified registry (hostname as image and port as tag).
+func stripRegistryTransport(url string) string {
```

Comment:
Maybe we need to export this function in order to use below:
pkg/extensions/sync/sync_test.go:763:                   registryName := strings.Replace(strings.Replace(srcBaseURL, "http://", "", 1), "https://", "", 1)
pkg/extensions/sync/sync_test.go:852:                   registryName := strings.Replace(strings.Replace(srcBaseURL, "http://", "", 1), "https://", "", 1)
pkg/extensions/sync/sync_test.go:918:                   registryName := strings.Replace(strings.Replace(srcBaseURL, "http://", "", 1), "https://", "", 1)
pkg/extensions/sync/sync_test.go:976:                   registryName := strings.Replace(strings.Replace(srcBaseURL, "http://", "", 1), "https://", "", 1)